kt/godot-runtime/src/main/kotlin/godot/runtime/Registration.kt

```diff
@@ -665,8 +666,8 @@ class ClassBuilderDsl<T : KtObject>(
 class ClassRegistry {
     val classes = mutableListOf<KtClass<*>>()
 
-    fun <T : KtObject> registerClass(name: String, superClass: String, isTool: Boolean = false, cb: ClassBuilderDsl<T>.() -> Unit) {
-        val builder = ClassBuilderDsl<T>(name, superClass)
+    fun <T : KtObject> registerClass(name: String, superClass: String, isTool: Boolean = false, baseGodotClass: String = superClass, cb: ClassBuilderDsl<T>.() -> Unit) {
```

Comment:
I wouldn't set the default value to the superclass. we cannot check that. better to be explicit in this case imo.